python/ribasim/ribasim/input_base.py

```diff
@@ -374,8 +374,9 @@ def _write_table(self, path: FilePath) -> None:
 
         gdf = gpd.GeoDataFrame(data=self.df)
         gdf = gdf.set_geometry("geometry")
+        gdf.index.name = "fid"
 
-        gdf.to_file(path, layer=self.tablename(), driver="GPKG")
+        gdf.to_file(path, layer=self.tablename(), driver="GPKG", index=True)
```

Comment:
Mostly out of curiosty: What happens if index=True but no name is provided, does it error?

Reply:
Oef, then it writes the index to an `index` column, and still adds a 1-based autoincrement fid.
This is the trivial model, which has edge_id 9 and 11 only.
![image](https://github.com/Deltares/Ribasim/assets/4471859/792c202e-f70e-493c-a840-1f6e6f4e54e6)